packages/node/src/configure/configure.module.ts

```diff
@@ -56,7 +56,14 @@ export class ConfigureModule {
     } else {
       if (!argv.subquery) {
         logger.error(
-          'subquery path is missing neither in cli options nor in config file',
+          'Subquery path is missing neither in cli options nor in config file',
+        );
+        yargsOptions.showHelp();
+        process.exit(1);
+      }
+      if (argv.subquery && argv.schema) {
```

Comment:
I think it is allowed, --schema overrides --subquery and --subquery-name

actually we can mark --subquery-name as deprecated. and encourage people to use --schema